src/main/java/mate/academy/rickandmorty/service/SaveCharacters.java

```diff
@@ -0,0 +1,21 @@
+package mate.academy.rickandmorty.service;
+
+import jakarta.annotation.PostConstruct;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import mate.academy.rickandmorty.dto.external.CharacterResponseDataDto;
+import org.springframework.stereotype.Component;
+
+@RequiredArgsConstructor
+@Component
+public class SaveCharacters {
+    private final CharacterService characterService;
+    private final RickAndMortyClient rickAndMortyClient;
+
+    @PostConstruct
+    public void saveCharacters() {
```

Comment:
method doesn't represent full specter of actions it's doing
```suggestion
    public void fetchCharactersToDb() {
```